Example/Tests/Tests.swift

```diff
@@ -208,24 +198,22 @@ class Tests: XCTestCase {
     func testReturningExpiredTokensFromTokenManager() {
         // Validate that mock token manager returns a null token
         let validTokensExpectation = expectation(description: "Will return tokens without errors")
-        TestUtils.tokenManagerNoValidationWithExpiration
-            .then { tokenManager in
-                XCTAssertEqual(tokenManager.accessToken, TestUtils.mockAccessToken)
-                XCTAssertEqual(tokenManager.idToken, TestUtils.mockIdToken)
-                XCTAssertEqual(tokenManager.refreshToken, TestUtils.mockRefreshToken)
-
-                // Wait 5 seconds for token to expire and update validation options
-                // to check for expiration
-                tokenManager.validationOptions["exp"] = true
-                sleep(5)
+        TestUtils.tokenManagerNoValidation
+        .then { tokenManager in
+            self.assertTokenManagerContents(tokenManager)
+            validTokensExpectation.fulfill()
+        }
+        .catch { error in XCTFail(error.localizedDescription) }
 
-                XCTAssertEqual(tokenManager.accessToken, nil)
-                XCTAssertEqual(tokenManager.idToken, nil)
-                validTokensExpectation.fulfill()
-            }
-            .catch { error in XCTFail(error.localizedDescription) }
-        
         waitForIt(10)
+
+        OktaAuth.tokens?.validationOptions["exp"] = true
+        // Wait for the token to expire and update validation options
+        // to check for expiration
+        sleep(5)
```

Comment:
As a follow-up task, I'm looking into a better way to jump ahead/back in time to ensure a token has expired. I strongly dislike using `sleep` functions in tests, but couldn't find a better alternative.